dask/dataframe/tests/test_groupby.py

```diff
@@ -2383,10 +2383,16 @@ def test_df_groupby_idxmin_skipna(skipna):
 
     ddf = dd.from_pandas(pdf, npartitions=2)
 
-    result_pd = pdf.groupby("group").idxmin(skipna=skipna)
     result_dd = ddf.groupby("group").idxmin(skipna=skipna)
 
-    assert_eq(result_pd, result_dd)
+    if not skipna and PANDAS_GE_210:
+        with pytest.warns(FutureWarning, match="all-NA values"):
+            result_pd = pdf.groupby("group").idxmin(skipna=skipna)
+        with pytest.warns(FutureWarning, match="all-NA values"):
+            assert_eq(result_pd, result_dd)
+    else:
+        result_pd = pdf.groupby("group").idxmax(skipna=skipna)
```

Comment:
Looks like this should be `idxmin` (there's a separate test for `idxmax`)

Reply:
Oh yes copy-paste